### ISSUE TITLE
permissive complexValue schema

### ISSUE THREAD

**Issue by redmitry** [ga4gh-beacon/beacon-v2#87]
Hi all,

There is a bug in the the [complexValue.json](https://github.com/ga4gh-beacon/beacon-v2/blob/e7d79620cccc3c93e9a559d9add3955491a6614c/models/json/beacon-v2-default-model/common/complexValue.json#L35) schema. 
```json
"properties": {
    "typedQuantities": {
        "description": "List of quantities required to fully describe the complex value",
        "items": {
            "$ref": "#/definitions/TypedQuantity"
        },
        "type": "array"
    }
},
```
The **typedQuantities** property is not required so **ANY** value would be accepted. 
This gives the problem in the [measurement.json](https://github.com/ga4gh-beacon/beacon-v2/blob/main/models/json/beacon-v2-default-model/common/measurement.json#L19)
```json
"measurementValue": {
    "description": "The result of the measurement",
    "oneOf": [
        {
            "$ref": "./value.json"
        },
        {
            "$ref": "./complexValue.json"
        }
    ]
},
```
The problem is that **IF** some value validates over **value.json**, as it also validates over **complexValue.json** validations fails, as it must validate only with one schema in "**oneOf**".

The solution is to put `"required": [ "typedQuantities" ] `in the complexValue.

Cheers,

Dmitry

**Comment by mbaudis** [comment]
Correct; also so defined in Phenopackets which served as a donor: https://phenopacket-schema.readthedocs.io/en/latest/complex-value.html#rstcomplexvalue 

I guess this is one of the instances where the "have to transcribe from protobuf & docs" bites us.

Pls. coordinate w/ @costero-e for the PR :-)

**Comment by costero-e** [comment]
This issue has been addressed by the PR #89 . I proceed to close this issue and continue conversation at PR.
